packages/stryker-karma-runner/test/integration/KarmaTestRunner.it.ts

```diff
@@ -238,4 +238,33 @@ describe('KarmaTestRunner', () => {
       return true;
     }));
   });
+
+  describe('when specified port is not available', () => {
+
+    before(() => {
+      const testRunnerOptions: RunnerOptions = {
+        fileNames: [],
+        port: 9882,
```

Comment:
I can see this test actually works, because it picks the same port as previous integration test. However, i would like to keep the integration tests independent. Could open a server ourselves and verify that a different port is picked?

Reply:
Yeah sure